### DISCUSSION TITLE
I use px-5 and px-10 at same class but px-10 is not applied

### DISCUSSION THREAD

**Discussion by NhaPCS** [tailwindlabs/tailwindcss#9101] [Help]
Hi, I haven't gotten this issue before, my code is not changed, but in some way, I use `px-5` and `px-10` in the same class (please check the photo below). 

The `px-10` should be overrides `px-5` but it's not. It overrides before.

<img width="473" alt="Screen Shot 2022-08-15 at 16 02 54" src="https://user-images.githubusercontent.com/9054422/184608021-7210b8b0-1ac0-4601-88b1-85c9f4e13f27.png">

Has anyone face this issue before? 
Thanks for your help!

**Comment by wongjn** [comment]
Could you let us know more about your setup? Also you shouldn't need to have both `px-5` and `px-10` on the same element anyway.

**Reply by NhaPCS** [reply]
Hi, @wongjn my situation like: `px-5` is the default class, and `px-10` is a custom class, so in some cases, it will have the same multiple in a class.

But I tried with `px-5 px-20`, `px-20` overrides the `px-5`, I don't understand `px-10` not.

It works before.

**Reply by wongjn** [reply]
Could you let us know more about your project setup? Also you shouldn't need to have both `px-5` and `px-10` on the same element anyway since they both have the same properties in the same context.

**Reply by NhaPCS** [reply]
My project is nextjs, I setup tailwind with postcss and autoprefixer @wongjn

**Comment by adamwathan** [comment]
These two classes conflict. Our IntelliSense plugin will warn you if you do this, it’s like adding `grid` and `flex` to an element at the same time. With the JIT engine, conflicting classes aren’t guaranteed to be generated in a particular order, it depends on which one Tailwind happens to see first in your content. Right solution here is to not add conflicting classes, make sure only one of them is actually on the element at a time.
